### PR TITLE
Add parameters for checking start state

### PR DESCRIPTION
**Pull request by livanov93**
This PR comes in combination with [ros2_controlPR128](https://github.com/ros-controls/ros2_control_demos/pull/128).
The idea is to disable execution of test scripts if the start state check has not satisfied configured limits.

### CODE REVIEW

**Review by destogl — APPROVED** [review]
This looks good! Thanks!

**Comment by destogl** [comment]
Waiting for
- ros-controls/ros2_control_demos#128
before merge.

File: ur_bringup/config/test_goal_publishers_config.yaml [hunk 1]
```diff
@@ -18,6 +18,14 @@ publisher_scaled_joint_trajectory_controller:
       - wrist_2_joint
       - wrist_3_joint
 
+    check_starting_point: true
+    starting_point_limits:
+      shoulder_pan_joint: [-0.1,0.1]
+      shoulder_lift_joint: [-1.6,-1.5]
+      elbow_joint: [-0.1,0.1]
+      wrist_1_joint: [-0.1,0.1]
```
Comment (MGBla) [inline ur_bringup/config/test_goal_publishers_config.yaml:26]:
```suggestion
      wrist_1_joint: [-1.6,-1.5] 
```

File: ur_bringup/config/test_goal_publishers_config.yaml [hunk 2]
```diff
@@ -38,3 +46,12 @@ publisher_joint_trajectory_controller:
       - wrist_1_joint
       - wrist_2_joint
       - wrist_3_joint
+
+    check_starting_point: true
+    starting_point_limits:
+      shoulder_pan_joint: [-0.1,0.1]
+      shoulder_lift_joint: [-1.6,-1.5]
+      elbow_joint: [-0.1,0.1]
+      wrist_1_joint: [-0.1,0.1]
```
Comment (MGBla) [inline ur_bringup/config/test_goal_publishers_config.yaml:55]:
```suggestion
      wrist_1_joint: [-1.6,-1.5] 
```

**Review by MGBla — CHANGES_REQUESTED** [review]
Not sure if the single models differ but on my ur3e i had a different home pose for wrist_1_joint. Otherwise tested it on the real robot and works as expected.

**Review by urrsk — CHANGES_REQUESTED** [review]
Good job.
I can approve with @MGBla correction of wrist_1_joint start angle position

File: ur_bringup/config/test_goal_publishers_config.yaml (hunk 2 above)
Comment (livanov93) [reply ur_bringup/config/test_goal_publishers_config.yaml]: Resolved with [https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/143/commits/13e2cbad11c9f02dbf158fc5795af6817dee42fa](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/143/commits/13e2cbad11c9f02dbf158fc5795af6817dee42fa)

File: ur_bringup/config/test_goal_publishers_config.yaml (hunk 1 above)
Comment (livanov93) [reply ur_bringup/config/test_goal_publishers_config.yaml]: Resolved with [https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/143/commits/13e2cbad11c9f02dbf158fc5795af6817dee42fa](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/143/commits/13e2cbad11c9f02dbf158fc5795af6817dee42fa)

**Review by urrsk — APPROVED** [review]

**Review by MGBla — APPROVED** [review]
LGTM

**Pull request merged**